tests/zarrLoader.spec.js

```diff
@@ -5,143 +5,130 @@ import { zeros, NestedArray } from 'zarr';
 import { ZarrLoader } from '../src/loaders';
 
 test('Test zarr non-rgb image loader', async t => {
-  try {
-    const z = await zeros([2, 500, 250], {
```

Comment:
Any reason for removing the `try` / `catch`? I believe you need to manually throw an error if something goes wrong in the async block.

Reply:
`tape-catch` should handle failing tests properly.  I’ll test it out again to be sure.